docs/references/api/swagger.json

```diff
@@ -1,5 +1,15 @@
-{ "info": {"title": "Epinio", "version":"1"},
+{
+  "schemes": [
+    "http",
+    "https"
+  ],
   "swagger": "2.0",
+  "info": {
+    "description": "This is the API specification of the Epinio API",
+    "title": "Epinio API",
+    "version": "1.0.0"
+  },
+  "basePath": "/api/v1",
```

Comment:
What swagger version does generate all this ?
I have 0.28.0 and it does not generate `info`. 
Which is why the Makefile contained `sed -i 's/^{/{ "info": {"title": "Epinio", "version":"1"},/' docs/references/api/swagger.json` as post processing after `swagger generate`.

Reply:
Oh! Ok.